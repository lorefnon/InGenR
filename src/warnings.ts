export interface WarningEntry {
  index?: number
  message: string
  error?: Error
}

type WarningEntryFactory = (...args: any[]) => WarningEntry

export const warnInvalidArgBody = () => ({
  message:
    "Detected invalid line in the arguments body of an InGenR directive.\n" +
    "This line will be discarded."
})

