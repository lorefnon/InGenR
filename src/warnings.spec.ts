import { warnInvalidArgBody } from "./warnings";

describe('warnings', () => {
    describe('warnInvalidArgBody', () => {
        it('works as expected', () => {
            expect(warnInvalidArgBody()).toMatchSnapshot()
        })
    })
})
