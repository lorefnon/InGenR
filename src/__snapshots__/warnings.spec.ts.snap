// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`warnings warnInvalidArgBody works as expected 1`] = `
Object {
  "message": "Detected invalid line in the arguments body of an InGenR directive.
This line will be discarded.",
}
`;
